src/views/product-tutorials-view/components/product-view-content/components/branded-callout/index.tsx

```diff
@@ -45,6 +45,12 @@ function BrandedCallout({
 				'--gradient-stop': `var(--token-color-${product}-gradient-faint-stop)`,
 		  }
 		: gradientDefault
+
+	const isExternal =
+		typeof location !== 'undefined' &&
+		cta.url.startsWith('http') &&
+		!cta.url.startsWith(location.origin)
```

Comment:
🙋🏻 Is it possible to use our `isAbsoluteUrl` helper here?

Reply:
`isAbsoluteUrl` doesn't appear to differentiate between an _internal/external_ link, so it isn't particularly helpful in this situation

Reply:
We use it that way in several places throughout the app, did you find a particular case where it doesn't work in that way?

Reply:
In this situation, I'd need to open:
- https://developer.hashicorp.com in the same tab
- https://www.hashicorp.com in a new tab

`isAbsoluteUrl` returns `true` for both of these.

---

If you're suggesting/asking if I can replace the following:
```diff
- cta.url.startsWith('http')
+ isAbsoluteUrl(cta.url)
```
that's certainly an option

Reply:
Got it! This component can receive input that comes from a remote source too, so that makes sense why we can't say:

- absolute URL = external
- non-absolute URL = internal

Other uses like sidebar have made it possible to make these assumptions.

---

💭 What do you think about moving the `isExternal` check into its own file in `src/lib/`? I can see us benefitting from either updating `isAbsoluteUrl` to use it, or replace references to `isAbsoluteUrl` with the new helper!

Reply:
For a reusable `isExternal` helper, I'd like to leverage the `host` value that we're grabbing in `_app` `.getInitialProps`, but I suspect that will be a more involved effort, so my thinking is to do that in a follow up PR

https://github.com/hashicorp/dev-portal/blob/cc3161acb52f08b35135664e166a0334093cfd01/src/pages/_app.tsx#L63-L64